debug/index.html

```diff
@@ -125,7 +125,7 @@
         },
         "osrm": {
             "type": "vector",
-            "tiles" : ["http://router.project-osrm.org/tile/v1/car/tile({x},{y},{z}).mvt"]
+            "tiles" : ["http://localhost:5000/tile/v1/car/tile({x},{y},{z}).mvt"]
```

Comment:
that's ok, the PR should not be merged and should be used only for comments. I specially made it to use a local server and will close the PR after the workshop without merging.